### PR TITLE
WatchFacePineTimeStyle: Simplify time update check

### PR DESCRIPTION
**Pull request by Riksu9000**
This isn't as thorough as the other PRs, because simply using another time point to check if a minute has passed would've used more memory.

### CODE REVIEW

File: src/displayapp/screens/WatchFacePineTimeStyle.cpp [hunk 1]
```diff
@@ -480,21 +477,13 @@ void WatchFacePineTimeStyle::Refresh() {
...
-    if (displayedSecond != second) {
-      displayedSecond = second;
-      lv_label_set_text_fmt(timeDD3, ":%02d", second);
-    }
-
-    if ((year != currentYear) || (month != currentMonth) || (dayOfWeek != currentDayOfWeek) || (day != currentDay)) {
+    currentDate = std::chrono::time_point_cast<days>(currentDateTime.Get());
+    if (currentDate.IsUpdated()) {
       lv_label_set_text_static(dateDayOfWeek, dateTimeController.DayOfWeekShortToString());
+      auto day = dateTimeController.Day();
```
Comment (FintasticMan) [inline src/displayapp/screens/WatchFacePineTimeStyle.cpp:483]: Don't use auto for integers.

File: src/displayapp/screens/WatchFacePineTimeStyle.h [hunk 2]
```diff
@@ -42,22 +42,19 @@ namespace Pinetime {
       private:
         uint8_t displayedHour = -1;
         uint8_t displayedMinute = -1;
```
Comment (FintasticMan) [inline src/displayapp/screens/WatchFacePineTimeStyle.h:44]: I would say it's better to use `UINT8_MAX` here instead of relying on underflow.

**Review by FintasticMan — CHANGES_REQUESTED** [review]
This looks alright, but I think the question here is whether consistency and readability is more important than a slight decrease in memory usage. I haven't tested it myself, but I assume the increase from using another time_point isn't very much, seeing as it decreases memory usage in other watch faces.

File: src/displayapp/screens/WatchFacePineTimeStyle.h (hunk 2 above)
Comment (Riksu9000) [reply src/displayapp/screens/WatchFacePineTimeStyle.h:44]: There's no guideline about this, so it's more subjective and not really in the scope of this PR.

**Review by NeroBurner — APPROVED** [review]
LGTM

**Pull request closed without merging**